documentation/src/main/asciidoc/topics/proc_restarting_clusters.adoc

```diff
@@ -0,0 +1,13 @@
+[id='restarting_clusters-{context}']
+= Restarting {brandname} Clusters
+Bring {brandname} clusters back online after shutting them down.
+
+If you shutdown clustered nodes with the [command]`shutdown server` command, you must restart each server in reverse order. +
+For example, if you shutdown `server1` and then shutdown `server2`, you should first start `server2` and then start `server1`.
+
+If you shutdown a cluster with the [command]`shutdown cluster` command, clusters become fully operational only after all nodes rejoin. +
+You can restart nodes in any order but the cluster remains in DEGRADED state until all nodes that were joined before shutdown are running.
+
+.Procedure
+
+* Start each node in the cluster with the `server` script.
```

Comment:
what is the `server` script?

Reply:
`bin/server.sh` Maybe there is a better name for it but we refer to it like that elsewhere in the server docs.

Reply:
@pruivo I just reworked this topic because I wasn't happy with it in the first place. That procedure seems a little useless tbh so I changed it to a concept to explain what users need to know when restarting clusters.